cmd/coverage/main_test.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"flag"
+	"testing"
+)
+
+func Test_Main(t *testing.T) {
```

Comment:
This should be MixedCaps: https://golang.org/doc/effective_go.html#mixed-caps

If you do use `TestMain()` note that it does have special intentions: https://golang.org/pkg/testing/#hdr-Main

Mainly (no pun intended), `TestMain()` is used for setup and tear-down of tests, and other test cases won't be called directly.

Reply:
Good point! I think I will remove the test, since `Main` has no immediately testable data returned. 

Instead, for now, I think I will have the tool run as a CI step so it's easy to view there as more of a one-off thing. If it's helpful to make it _more_ evident and usable down the line, I could also later add it to a local file and reference it in the README.